modules/marketo_ma_contact/src/Form/MarketoMaContactConfiguration.php

```diff
@@ -77,8 +77,8 @@ public function buildForm(array $form, FormStateInterface $form_state, ContactFo
 
     $form['enabled'] = [
       '#type' => 'checkbox',
-      '#title' => $this->t('Enable tracking'),
-      '#description' => $this->t('Set to "Enable tracking" to turn on tracking for this contact form.'),
+      '#title' => $this->t('Capture Data'),
```

Comment:
"Capture Data" is the terminology that was used in the documentation. Did we move away from this terminology for a reason. I think the description could be better for sure:

> This needs to be checked in order to enable data capture for this contact form.